File: jifrozen/정렬/k번째수.java
package 정렬;
import java.util.*;
class k번째수 {
	public int[] solution(int[] array, int[][] commands) {
		int[] answer = new int[commands.length];
		for(int i=0;i<commands.length;i++){
			ArrayList<Integer> arr=new ArrayList<>();

			int a=commands[i][0];
			int b=commands[i][1];
			int c=commands[i][2];
			for(int j=a-1;j<b;j++){
				arr.add(array[j]);
			}
			Collections.sort(arr);
			answer[i]=arr.get(c-1);
		}
		return answer;
	}
}

File: yunva17/정렬/K번째수.py
array = [1, 5, 2, 6, 3, 7, 4]
commands = [[2, 5, 3], [4, 4, 1], [1, 7, 3]]

def solution(array, commands):
    answer = []
    
    for i in commands:
        ans = array[i[0]-1:i[1]]
        ans.sort()
        answer.append(ans[i[2]-1])

    return answer

print(solution(array,commands))